server/src/main/java/org/elasticsearch/rest/action/document/RestBulkAction.java

```diff
@@ -117,10 +120,16 @@ public RestChannelConsumer prepareRequest(final RestRequest request, final NodeC
                     request.getRestApiVersion()
                 );
             } catch (Exception e) {
+                content.close();
```

Comment:
We shouldn't close the content on exception here any more.

Reply:
oops, thank you!